bin/cqlsh.py

```diff
@@ -2169,14 +2180,15 @@ def read_options(cmdlineargs, environment):
         credentials.read(options.credentials)
 
         # use the username from credentials file but fallback to cqlshrc if username is absent from the command line parameters
-        options.username = option_with_default(credentials.get, 'plain_text_auth', 'username', username_from_cqlshrc)
+        options.username = username_from_cqlshrc
```

Comment:
@bhouse99 I do not get this change. So we are populating `options.username` with `username_from_cqlshrc`. So for what reason do we read `credentials.read(options.credentials) 3 lines above? That seems to me like dead code?